addons/slack/uninstall.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Stylistically novel comment brackets used here.

Reply:
Won't fix, it's there in a number of template files, including the other two checked-in addons